src/scripts/reducers/layers/details.ts

```diff
@@ -13,6 +15,15 @@ function detailsReducer(
 ): DetailsById {
   switch (action.type) {
     case FETCH_LAYER_SUCCESS:
+      if (action.id === 'land_cover.lccs_class') {
```

Comment:
If the layer data is in the layers metadata it would be already in the state.
And even it is not there's no need to add it here to the state. You could just import it in the component directly when the id is 'land-cover'.